test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1575,6 +1642,8 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
         return setupSocial()
         .then(function (test) {
           test.form.facebookButton().click();
+          return tick(test);
+        }).then(function(test) {
```

Comment:
This will not work with **waitForSpyCall**. Not sure why. Sticking with **tick()** here.